xbmc/pictures/Picture.cpp

```diff
@@ -161,24 +161,30 @@ bool CPicture::ResizeTexture(const std::string &image, uint8_t *pixels, uint32_t
   // create a buffer large enough for the resulting image
   GetScale(width, height, dest_width, dest_height);
 
-  uint8_t *buffer = new uint8_t[dest_width * dest_height * sizeof(uint32_t)];
+  // Let's align so that stride is always dividable by 16 and some 32 byte more on top
```

Comment:
The term "divisible" would be more correct that "dividable" here. Any two numbers are dividable, but divisible specifically means that they have zero remainder when divided.

How about:

```c++
// Let's align so that stride is always divisible by 16, and then add some 32 bytes more on top
```

Reply:
Sounds better. I updated it.